### PR TITLE
VACC Estonia updates (AIRAC 2307)

### PR DESCRIPTION
**Pull request by mjpikat**
- Renamed "Tallin" to "Tallinn" in EETN.json;
- Added sector EETT_H_APP (Tallinn Control, INTOR Feeder sector);
- Added sector EETU_I_TWR (Tartu Information) as its shape is TMA-size.

**All the sectors match current GNG setup**

### CODE REVIEW

**Comment by Soofushka** [comment]
@mjpikat You did a great job! EETU_I_TWR boundaries, unfortunately, wiill not be displayed, you will only see something like this:
image

File: Boundaries/EETU/EETU_I.json
Comment (dirtyformal) [inline Boundaries/EETU/EETU_I.json:1]: Hi there, thanks for contributing. Please note that this project is only for TRACON/TMA boundaries, and not for Towers (even if providing a Procedural service). Please remove this, and then I'll review.

Thanks!

**Review by dirtyformal — CHANGES_REQUESTED** [review]
As per my comment on EETU_I.json.

File: Boundaries/EETU/EETU_I.json
Comment (mjpikat) [reply Boundaries/EETU/EETU_I.json:1]: > Hi there, thanks for contributing. Please note that this project is only for TRACON/TMA boundaries, and not for Towers (even if providing a Procedural service). Please remove this, and then I'll review.
> 
> Thanks!

Hi Tom! Thanks for the heads-up, I have adjusted the commit now :)

M-J

**Review by kmoberg — APPROVED** [review]

**Pull request merged**